spyder/widgets/editor.py

```diff
@@ -1191,6 +1198,18 @@ def close_all_but_this(self):
         self.close_all_right()
         for i in range(0, self.get_stack_count()-1  ):
             self.close_file(0)
+            
+    def add_last_closed_file(self, fname):
+        """Add to last closed file list"""

```

Comment:
Docstrings are sentences so a `.` is needed at the end :-)

`"""Add to last closed file list."""`